enterprise/server/cmd/server/main.go

```diff
@@ -104,7 +105,7 @@ func convertToProdOrDie(ctx context.Context, env *real_environment.RealEnv) {
 	if err == nil {
 		env.SetAuthenticator(authenticator)
 	} else {
-		log.Infof("No authentication will be configured: %s", err)
+		alert.UnexpectedEvent("authentication_configuration_failed", "Failed to configure authentication: %s", err)
```

Comment:
we alert on this metric (it will send a page). Is this really an unexpected event?

Reply:
Moved the alert to where we actually try to configure at least one issuer, but it fails.